embedchain/config/apps/AppConfig.py

```diff
@@ -16,16 +16,23 @@ class AppConfig(BaseAppConfig):
     Config to initialize an embedchain custom `App` instance, with extra config options.
     """
 
-    def __init__(self, log_level=None, host=None, port=None, id=None):
+    def __init__(self, log_level=None, host=None, port=None, id=None, db_type=None):
         """
         :param log_level: Optional. (String) Debug level
         ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'].
         :param host: Optional. Hostname for the database server.
         :param port: Optional. Port for the database server.
         :param id: Optional. ID of the app. Document metadata will have this id.
+        :param db_type: Optional. db type to use. Currently [chroma, es] are supported.
         """
         super().__init__(
-            log_level=log_level, embedding_fn=AppConfig.default_embedding_function(), host=host, port=port, id=id
+            log_level=log_level,
+            embedding_fn=AppConfig.default_embedding_function(),
+            host=host,
+            port=port,
+            id=id,
+            db_type=db_type,
+            vector_dim=1536,  # vector length created by embedding fn
```

Comment:
rather than writing 1536, we should refer it via a variable.
maybe maintain a constants file, where you can store variables along with their values.
this way if the value changes, we just need to update at one place only.

Reply:
update: referring from VectorDimensions under providers